contrib/babelfishpg_common/sql/smalldatetime.sql

```diff
@@ -489,6 +494,14 @@ DEFAULT FOR TYPE sys.SMALLDATETIME USING hash AS
     OPERATOR    1   =  (sys.SMALLDATETIME, sys.SMALLDATETIME),
     FUNCTION    1   smalldatetime_hash(sys.SMALLDATETIME);
 
+ALTER OPERATOR FAMILY sys.smalldatetime_ops USING btree ADD
```

Comment:
This may not work if you have index on `date` column and predicate involves smalldatetime column. Please check if we have such test cases? and add if there is not already

Reply:
Yes, if we create index on date column and compare with smalldatetime will not choose index clause as their is not operator family for date. Added those test cases.